src/Controls/src/Core/Platform/AlertManager/AlertManager.Windows.cs

```diff
@@ -32,6 +32,7 @@ internal AlertRequestHelper(Window virtualView, UI.Xaml.Window platformView)
 
 			public UI.Xaml.Window PlatformView { get; }
 
+			// TODO: This method is obsolete in .NET 10 and will be removed in .NET11.
```

Comment:
Why dont add ?

```
[Obsolete("This method is obsolete in .NET 10 and will be removed in .NET11.")]
```

Reply:
@rmarinho, I have already marked this method as `Obsolete` in the shared class (**[AlertRequestHelper](https://github.com/dotnet/maui/blob/9be40fbae67e9de433e550bc8072c27a90e79c80/src/Controls/src/Core/Platform/AlertManager/AlertManager.cs#L89)**), where the partial method is declared. If I also add Obsolete in the platform-specific implementation, it results in a `duplicate attribute error`, which is why I added a comment instead.

Could you please confirm if it’s acceptable to mark this method as Obsolete in the shared declaration, or should it be applied only in the platform-specific implementation instead?